insights/misc/importer.py

```diff
@@ -30,17 +30,18 @@ def import_repo(request, pk):
         log.append("Saved workflow: %s" % workflow.path)
 
     for workflow in Workflow.objects.filter(repo=repo):
-        workflow_from_api = repo_from_api.get_workflow(workflow.path)
+        workflow_from_api = repo_from_api.get_workflow("%s" % workflow.workflow_id)
```

Comment:
```suggestion
        workflow_from_api = repo_from_api.get_workflow(str(workflow.workflow_id))
```

Will cast it as a string and is a little more idiomatic.